CryptKeeperZK/crypt-keeper-extension
# feat: add activity ui page

## Explanation

This PR adds support for Activity tab.

Details are below:
- [x] Add track for insert, delete operations
- [x] Integration fixes for new operation type and settings
- [x] Add sort by date for operations
- [x] Add ActivityList component

## More Information

Blocked by #254 
Closes #30

## Screenshots/Screencaps

### Before

N/A

### After

![image](https://user-images.githubusercontent.com/14254374/232169192-632bf0bc-c45d-4b79-9bb8-f894313251b4.png)

## Manual Testing Steps

1. Switch to activity tab
2. Create/delete identity or remove all identities
3. See changes on Activity tab
4. Remove one history operation or remove all of them
5. Enable/disable "Keep track history" checkbox

## Pre-Merge Checklist


### + If there are functional changes:


> PR template source from [github.com/MetaMask](https://github.com/MetaMask)
